src/components/organisms/Join/index.jsx

```diff
@@ -296,17 +289,6 @@ const Join = () => {
                 paddingVariant={buttonPaddingVariants.SYMMETRIC}
               />
             </RecommendationCardDescriptionWrapper>
-            <picture>
-              <source srcSet={appleWatchAvifPath} type="image/avif" />
-              <source srcSet={appleWatchWebpPath} type="image/webp" />
-              <AppleWatchImg
-                src={appleWatchPngPath}
-                alt={t('join.appleWatchImageAlt')}
-                width="80"
-                height="94"
-                loading="lazy"
-              />
-            </picture>
```

Comment:
If you remove this you should remove `margin-right` on `RecommendationCardDescriptionWrapper`